packages/containers/src/AppLoader/AppLoader.md

```diff
@@ -0,0 +1,45 @@
+# App Loader
+
+This component is here to make http calls before the first render of the application. During the loading, it show the application's loader.
+
+## Where i put it ?
+
+On your top level component, you can place it this way :
+
+```html
+<AppLoader>
+    <div>
+        <ShortcutManager view="shortcuts" />
+        <IconsProvider icons={mockIconDefinition} />
+        <ACKDispatcher />
+        <Notification />
+        {props.children}
+    </div>
+</AppLoader>
+```
+
+## How to configure it ?
+
+This is a sample of configuration :
+
+```json
+"AppLoader#default":{
+    "saga": "appLoaderSaga",
+    "steps":[
+        { "actionCreators":["user:fetchIdentity"] },
+        { "waitFor":["identity"] },
```

Comment:
i have the feeling that the waitFor declaration is not really usefull ? can each block of actionCreators wait for the positive resolution of their predecessors ?

Reply:
Don't know if i can make it this way. Here if for `user:fetchIdentity` we have this piece of code :
```
function fetchIdentity() {
	return actions.http.get(API.identity, {
		cmf: {
			collectionId: 'identity',
		},
	});
}
```
When the actionCreator would be resolved, the http call would be launched but we are far away from having the identity's collection provisioned in the store.

Reply:
No they are redux actions so non blocking.
In fact we have the two cases, the idea here to be able to make some wait in steps.
But not required